### ISSUE TITLE
ubuntu 14.04, default hhvm pid file path incorrect

### ISSUE THREAD

**Issue by cybacolt** [saltstack-formulas/php-formula#65]
from /etc/init.d/hhvm:
PIDFILE=/var/run/hhvm/pid

from the php formula map php/ng/map.jinja:
('pid', '/var/run/hhvm.pid'),

this causes hhvm not to start by default unless you add to your pillar:
php:
  ng:
    lookup:
      hhvm:
        server:
          pid: '/var/run/hhvm/pid'

not sure if this is easily changable - other distros may break if your change the default pid pathing?

**Comment by gravyboat** [comment]
Seems like something that we should add to the `map.jinja` for each distro, then merge that with the pillar data.

**Comment by cybacolt** [comment]
yeah, it its mapping per distro, but the value is still wrong for 'Debian'

**Comment by gravyboat** [comment]
I'd say prep a PR for the debian fix then, could have been someone just copy pasta'd it in an effort to save time.
